atom/typed.py

```diff
@@ -63,9 +76,9 @@ class ForwardTyped(Typed):
     normal typed.
 
     """
-    __slots__ = ('resolve', 'args', 'kwargs')
+    __slots__ = ('resolve', 'args', 'kwargs', "optional")
```

Comment:
Single quotes to be consistent

Reply:
One day I will just `black` everything ... but you are right